### PR TITLE
Sept 2017

### PR DESCRIPTION
**Pull request by seanwestfall**

### CODE REVIEW

File: public/_data.json [hunk 1]
```diff
@@ -4,45 +4,45 @@
       "datetime": "Thursday August 31st, 2017 7pm",
       "date": "8/31/2017 19:00:00",
       "host": {
-        "address": "2401 Colorado Ave, Santa Monica, CA 90404",
-        "name": "Edmunds",
-        "misc": "Parking validation provided in Colorado Center (company parking lot).",
-        "url": "https://www.edmunds.com/"
+        "address": "2110 Broadway, Santa Monica, CA 90404",
+        "name": "TigerText",
+        "misc": "Nearby street parking and some spaces available in our lot",
```
Comment (ashwell) [inline public/_data.json]: in **their** lot?

Comment (seanwestfall) [reply public/_data.json]: So when we write these, we always write it from our perspective?

**Comment by seanwestfall** [comment]
@ashwell Thanks for the review. So more changes coming shortly. :+1:

File: public/_data.json [hunk 2]
```diff
@@ -4,45 +4,45 @@
...
       "date": "8/31/2017 19:00:00",
       "host": {
-        "address": "2401 Colorado Ave, Santa Monica, CA 90404",
-        "name": "Edmunds",
-        "misc": "Parking validation provided in Colorado Center (company parking lot).",
-        "url": "https://www.edmunds.com/"
+        "address": "2110 Broadway, Santa Monica, CA 90404",
+        "name": "TigerText",
+        "misc": "Nearby street parking and some spaces available in their lot",
+        "url": "https://www.tigertext.com/"
```
Comment (rouzbeh84) [inline public/_data.json]: should this be the link to their site or the jobs board that we link to in email/eventbrite?

File: public/_data.json [hunk 3]
```diff
@@ -4,45 +4,45 @@
...
+            "logo": "host-tigertext.png",
+            "url": "https://www.tigertext.com/"
           },
           {
-            "name": "Headspace",
-            "logo": "host-headspace.png",
-            "url": "https://www.headspace.com/join-us"
+            "name": "Signal Sciences",
+            "logo": "sponsor-signal-sciences.svg",
+            "url": "https://www.signalsciences.com/"
```
Comment (rouzbeh84) [inline public/_data.json]: same here, career link?

**Pull request merged**